jck/runtime.api/playlist.xml

```diff
@@ -28,6 +28,22 @@
 			<group>jck</group>
 		</groups>
 	</test>
+	<test>
+		<testCaseName>jck_target_folder_check</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>$(TEST_ROOT)$(D)jck$(D)jck_target_folder_check$(D)jck_target_folder_check.sh $(JCK_VERSION) $(JCK_ROOT) $(TEST_ROOT); \
+	$(TEST_STATUS)</command>
+		<levels>
+			<level>extended</level>
+		</levels>
+		<groups>
+			<group>jck</group>
+		</groups>
+		<platformRequirements>os.linux,arch.x86,bits.64</platformRequirements> 
+
```

Comment:
Could you please remove this extra line? Other than that, LGTM.

Reply:
Thanks. Does it look okay now? https://github.com/AdoptOpenJDK/openjdk-tests/blob/af607922f8965cf31e750f351263bde5a4d04654/jck/runtime.api/playlist.xml